src/packet.rs

```diff
@@ -648,12 +648,18 @@ impl PktNumSpace {
             crypto_open: None,
             crypto_seal: None,
 
-            crypto_stream: stream::Stream::new(std::u64::MAX, std::u64::MAX),
+            crypto_stream: stream::Stream::new(
+                0,
```

Comment:
Presumably, the Stream ID property is meaningless for this crypto stream. So that raises the question, what purpose does the ID have other than to be used for determine if the stream is bidi or uni? Maybe you have some future use in mind.

Reply:
I don't remember why I did this, though I changed it now to track bidi directly, as you say it makes more sense.